tf_prefix: ""
vel_limit_x: 0.03
vel_limit_y: 0.03
vel_limit_z: 0.01
acc_limit_y: 0.001
acc_limit_z: 0.001

vel_limit_mir: 0.1
vel_reference_mir: -0.025